src/generate.ts

```diff
@@ -17,10 +17,18 @@ type RequiredModelSubset = Pick<
 	'tables' | 'relationships' | 'synonyms'
 >;
 
+export type Options = {
+	noSerial?: boolean;
+	noMissingReferences?: boolean;
+};
+
 const trimNL = new TemplateTag(
 	replaceResultTransformer(/^[\r\n]*|[\r\n]*$/g, ''),
 );
 
+const replaceSerial = (s: string): string =>
+	s.replace(/^(?:big\s)?serial$/i, 'Integer');
```

Comment:
`Big Serial` should become `Big Integer` imo, so I would say just do a straight swap of `Serial` -> `Integer` and keep the size modifiers as-is